slither.config.json

```diff
@@ -0,0 +1,5 @@
+{
+  "detectors_to_exclude": "external-function",
+  "filter_paths": "@openzeppelin|contracts/test",
+  "exclude_informational": true
```

Comment:
Excluding informational for now, but there are some useful errors in this category which we should include later